app/controllers/answers_controller.rb

```diff
@@ -1,12 +1,36 @@
 class AnswersController < ApplicationController
+  before_action :find_category
   def index
+
   end
 
   def new
-    @category = Category.find(params[:category_id])
-    @word = @category.words.build
-    @choices = choices.all
+    @words =  @category.words.paginate(page:params[:page], per_page: 1)
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new
   end
 
+  def create
+    @lesson = Lesson.find(params[:lesson_id])
+    @answer = Answer.new(answer_params)
+    if @lesson.save
+      unless params[:page].empty?
+        redirect_to new_category_lesson_answer_url(@word.page: params[:page])
+      else
+        result = 0
+        @cateogyry.words.each do |answer|
+          if answer.choice.correct?
+            result += 1
+          end
+          @lesson.update_attributes(:result =>result, :is_completed => true)
+          @lesson.create_activity(user_id: current_user.id)
+      end
+      redirect_to category_lesson(@category, @lesson)
+    end
+  end
```

Comment:
![image](https://user-images.githubusercontent.com/35232348/105464596-cf8adc80-5ccc-11eb-8350-f3fbb97efbde.png)
After you figure out how to properly implement Pagination.
Clean and fix your code because you are missing an `end`.